src/main/java/org/gitlab4j/api/models/ProjectOfGroupFilter.java

```diff
@@ -0,0 +1,167 @@
+package org.gitlab4j.api.models;
+
+import org.gitlab4j.api.Constants.ProjectOrderBy;
+import org.gitlab4j.api.Constants.SortOrder;
+import org.gitlab4j.api.Constants;
+import org.gitlab4j.api.GitLabApiForm;
+
+/**
+ *  This class is used to filter Projects when getting lists of projects for a specified group.
+ */
+public class ProjectOfGroupFilter {
+
```

Comment:
@mdeknowis 
I should have mentioned this in an earlier comment, but simply forgot.  The name of this class would be more descriptive as ```GroupProjectFilter```.  I am going to go ahead and merge, but change the name during the release process.